applications/nrf5340_audio/src/bluetooth/le_audio.h

```diff
@@ -155,14 +166,15 @@ int le_audio_user_defined_button_press(enum le_audio_user_defined_action action)
 /**
  * @brief Get configuration for audio stream
  *
- * @param bitrate	Pointer to bitrate used
- * @param sampling_rate	Pointer to sampling rate used
+ * @param bitrate	Pointer to bitrate used, can be NULL
+ * @param sampling_rate	Pointer to sampling rate used, can be NULL
+ * @param pres_delay	Pointer to presentation delay used, can be NULL
  *
  * @return	0 for success,
  *		-ENXIO if the feature is disabled,
```

Comment:
The return codes are changed in this PR, so this is not valid anymore

Reply:
Added a line for -ENOTSUP and changed -ECANCELED to -ENXIO and added a LOG_WRN